web/src/components/organisms/Project/Content/ContentDetails/hooks.ts

```diff
@@ -271,124 +281,80 @@ export default () => {
-            modelId: currentModel.id,
-            schemaId: data.metaSchemaId,
-            fields: data.metaFields.map(field => ({
-              ...field,
-              type: field.type as SchemaFieldType,
-            })),
-          },
-        });
-        if (metaItem.errors || !metaItem.data?.createItem) {
-          Notification.error({ message: t("Failed to create item.") });
-          return;
-        }
-        metaItemId = metaItem?.data.createItem.item.id;
-        const item = await updateItem({
-          variables: {
-            itemId: data.itemId,
-            fields: data.fields.map(field => ({
-              ...field,
-              type: field.type as SchemaFieldType,
-            })),
-            metadataId: metaItemId,
-            version: currentItem?.version ?? "",
-          },
-        });
-        if (item.errors || !item.data?.updateItem) {
-          Notification.error({ message: t("Failed to update item.") });
-          return;
-        }
+    async ({ metaItemId, metaFields }: { metaItemId: string; metaFields: ItemField[] }) => {
+      const item = await updateItem({
+        variables: {
+          itemId: metaItemId,
+          fields: metaFields as ItemFieldInput[],
+          version: currentItem?.metadata?.version ?? "",
+        },
+      });
+      if (item.errors || !item.data?.updateItem) {
+        Notification.error({ message: t("Failed to update item.") });
+        return;
+      }
+      Notification.success({ message: t("Successfully updated Item!") });
+    },
+    [updateItem, currentItem?.metadata?.version, t],
+  );
+
+  const dateConvert = useCallback((type: FieldType, value?: ItemValue) => {
+    if (type === "Date") {
+      if (Array.isArray(value)) {
+        return (value as string[]).map(valueItem => (valueItem ? moment(valueItem) : ""));
       } else {
-        const item = await updateItem({
-          variables: {
-            itemId: data.metaItemId as string,
-            fields: data.metaFields.map(field => ({
-              ...field,
-              type: field.type as SchemaFieldType,
-            })),
-            version: currentItem?.metadata?.version ?? "",
-          },
-        });
-        if (item.errors || !item.data?.updateItem) {
-          Notification.error({ message: t("Failed to update item.") });
-          return;
-        }
+        return value ? moment(value as string) : "";
       }
+    } else {
+      return value;
+    }
+  }, []);
 
-      Notification.success({ message: t("Successfully updated Item!") });
+  const valueGet = useCallback(
+    (field: Field) => {
+      switch (field.type) {
+        case "Select":
+          return field.typeProperty?.selectDefaultValue;
+        case "Integer":
+          return field.typeProperty?.integerDefaultValue;
+        case "Asset":
+          return field.typeProperty?.assetDefaultValue;
+        default:
+          return dateConvert(field.type, field.typeProperty?.defaultValue);
```

Comment:
You should not check for the date type inside the default case, this may cause some bugs and make the code hard to read.

```suggestion
        case "Date":
          return dateConvert(field.typeProperty?.defaultValue);
        default:
          return field.typeProperty?.defaultValue;
```